scripts/install/run.sh

```diff
@@ -215,74 +205,74 @@ function local_install {
     echo "Finished: npm @spartacus:registry set back to https://registry.npmjs.org/"
 }
 
-function prestart_csr {
-    if [ -z "$CSR_PORT" ]; then
-        echo "Skipping prestart csr script"
+function build_csr {
+    if [ -z "${CSR_PORT}" ]; then
+        echo "Skipping csr app build"
     else
-        printh "Prestart setup for csr app"
-        ( cd ${INSTALLATION_DIR}/csr && yarn build --prod )
+        printh "Building csr app"
+        ( cd ${INSTALLATION_DIR}/spartacus && yarn build --prod )
     fi
 }
 
-function prestart_ssr {
-    if [ -z "$SSR_PORT" ]; then
-        echo "Skipping prestart ssr script"
+function build_ssr {
+    if [ -z "${SSR_PORT}" ]; then
+        echo "Skipping ssr app build"
     else
-        printh "Prestart setup for ssr app"
-        ( cd ${INSTALLATION_DIR}/ssr && yarn build && yarn build:ssr )
+        printh "Building ssr app"
+        ( cd ${INSTALLATION_DIR}/spartacus-ssr && yarn build && yarn build:ssr )
```

Comment:
Can we get rid of this hardcoded app name and use param instead?

Reply:
Same as above